drivers/can/can_mcp2515.c

```diff
@@ -209,7 +209,7 @@ static int mcp2515_cmd_read_rx_buffer(const struct device *dev, uint8_t nm,
 	return spi_transceive_dt(&dev_cfg->bus, &tx, &rx);
 }
 
-static void mcp2515_convert_zcanframe_to_mcp2515frame(const struct zcan_frame
+static void mcp2515_convert_zcanframe_to_mcp2515frame(const struct can_frame
```

Comment:
I'd suggest to rename the function as well. Also the one for conversion in the other direction.

Reply:
Renamed, thanks.